src/components/ComposedChart.jsx

```diff
@@ -103,9 +103,21 @@ export default class ComposedChart extends BaseChart {
             }
         });
 
+        const legendColumns = Math.floor(width / 160);
+        const maxLegendItems = Math.floor((height - 100) / 25);
+        const legendOffset = config.legend === true && finalLegend.length > maxLegendItems ?
+            (((Math.ceil(finalLegend.length / legendColumns)) * 30) + 50) : 50;
```

Comment:
shouldn't we need to document these magic numbers somewhere?

Reply:
Fixed with https://github.com/wso2/react-vizgrammar/pull/207